README.md

```diff
@@ -1,2 +1,47 @@
-# zipkin-sparkstreaming-example
-Example of how to make a custom adjuster for zipkin-sparkstreaming
+# zipkin-sparkstreaming custom adjuster example
+
+This example shows how to add a custom adjuster for zipkin-sparkstreaming.
+We'll create a jar with a simple adjuster that trims the length of binary annotation values.
+This adjuster will be applied by placing it in the class path of the spark job.
+
+## Steps for creating an adjuster jar
+
+1. Create a java source file that..
+  * extends `zipkin.sparkstreaming.Adjuster`
+  * has an annotation `org.springframework.context.annotation.Configuration`
+2. Create a resource file `META-INF/spring.factories` that contains
+```
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+your.package.FooAdjuster
+```
+3. Put compiled class and `META-INF/spring.factories` into a jar.
+
+In maven, the following structure would accomplish this.
+```
+src/main/java/your/package/FooAdjuster.java
```

Comment:
in other words, I think this whole section can be taken out and replaced with a section that says that this project is an example setup that creates an adjuster jar.